pom.xml

```diff
@@ -11,26 +11,22 @@
     <description>Main Repo holding the sub modules for the generation of documents</description>
 
     <url>http://maven.apache.org</url>
-
     <parent>
         <groupId>uk.gov.companieshouse</groupId>
-        <artifactId>companies-house-spring-boot-parent</artifactId>
-        <version>1.0.0-rc1</version>
-        <relativePath/> <!-- lookup parent from repository -->
+        <artifactId>companies-house-parent</artifactId>
+        <version>2.1.6</version>
+        <relativePath/>
     </parent>
 
     <properties>
-        <!-- Encoding -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+       <jdk.version>21</jdk.version>
 
-        <!-- JDK Version -->
-        <jdk.version>1.8</jdk.version>
-
-        <!-- Maven and Surefire plugins -->
-        <maven-compiler-plugin.version>3.5.1</maven-compiler-plugin.version>
-        <maven-surefire-plugin.version>2.21.0</maven-surefire-plugin.version>
-        <maven-jar-plugin.version>2.4</maven-jar-plugin.version>
-        <junit-platform-surefire-provider.version>1.2.0</junit-platform-surefire-provider.version>
+        <maven-compiler-plugin.version>3.13.0</maven-compiler-plugin.version>
+        <maven-surefire-plugin.version>3.3.1</maven-surefire-plugin.version>
+        <maven-jar-plugin.version>3.4.1</maven-jar-plugin.version>
+        <junit-platform-surefire-provider.version>1.3.2</junit-platform-surefire-provider.version>
 
         <!-- Repositories -->
```

Comment:
Think these are defined in the parent pom now

Reply:
Yes , its parent POM.